### PR TITLE
fixed uui issue and npm audit

### PR DESCRIPTION
**Pull request by kdcllc**
there were issues running sample locally with uuid in service

@dluc please review this pr

### CODE REVIEW

File: semantic-workbench/v1/app/package-lock.json
Comment (dluc) [inline semantic-workbench/v1/app/package-lock.json:1]: do we need the changes in this file? unless there are changes also in `package.json` I think we can keep the original file

Comment (kdcllc) [reply semantic-workbench/v1/app/package-lock.json:1]: @dluc I had to run `npm audit fix` to fix critical issues with packages

File: examples/dotnet-example02/appsettings.json [hunk 1]
```diff
@@ -28,9 +28,9 @@
   },
   // Azure Content Safety settings
   "AzureContentSafety": {
-    "Endpoint": "https://....cognitiveservices.azure.com/",
+    "Endpoint": "https://openai1-x3boiusjqblbm.openai.azure.com/",
     "AuthType": "ApiKey",
-    "ApiKey": "..."
+    "ApiKey": "616c3edc5550418f801a27c90d6c01de"
```
Comment (dluc) [inline examples/dotnet-example02/appsettings.json:33]: please undo these changes -- the key will remain in the repo history even after updating the PR, so I recommend invalidating it ASAP

**Review by dluc — CHANGES_REQUESTED** [review]
- undo changes to package-lock.json
- undo changes to appsettings.json
- invalidate personal API key

**Pull request closed without merging**